src/client/opamAuxCommands.ml

```diff
@@ -504,7 +504,7 @@ let check_and_revert_sandboxing root config =
   | [] -> config
   | cmd::_ ->
     (* All the provided sandboxing scripts are expected to define [TMPDIR] *)
-    let test_file = "$TMPDIR/opam-sandbox-check-out" in
+    let test_file = "${TMPDIR:-/tmp}/opam-sandbox-check-out" in
```

Comment:
The default sandbox forbids writes to `/tmp` so this would fail anyway.

Reply:
Yes, but I think it's a good hardening regardless - at least there's no possible situation where we attempt to write to `/`